prov/efa/src/efa_av.c

```diff
@@ -1075,3 +1075,97 @@ int efa_av_open(struct fid_domain *domain_fid, struct fi_av_attr *attr,
 	free(av);
 	return ret;
 }
+
+#ifdef EFA_UNIT_TEST
+
+#include "efa_unit_tests.h"
+
+/* Common Setup/Teardown */
+
+struct efa_resource {
+	struct fid_fabric *fabric;
+	struct fid_domain *domain;
+	struct fid_av *av;
+};
+
+struct efa_resource *create_efa_resource() {
+    struct efa_resource *resource = (struct efa_resource*) malloc(sizeof(struct efa_resource));
+    resource->fabric = NULL;
+    resource->domain = NULL;
+    resource->av = NULL;
+
+    struct fi_info *info;
+    int rv = fi_getinfo(FI_VERSION(1, 14), NULL, NULL, 0ULL, NULL, &info);
```

Comment:
We typically use "ret" for return value, would be a little more consistent to continue doing so

Reply:
changed rv to ret globally.